backend/app/core/cache.py

```
@@ -203,7 +202,9 @@
 cache_manager = AgentCacheManager()
 
 
-def cached_response(cache_key_prefix: str, ttl: int = None, vary_by: list = None):
+def cached_response(
+    cache_key_prefix: str, ttl: int = None, vary_by: list = None
```

Ruff (0.12.2)

PEP 484 prohibits implicit `Optional`

Convert to `Optional[T]`

(RUF013)